strider/storage.py

```diff
@@ -6,7 +6,7 @@
 
 from .config import settings
 
-if settings.redis_url == 'redis://fakeredis':
+if settings.redis_url.startswith('redis://fakeredis'):
```

Comment:
Is this change related to the others?

Reply:
Kind-of. When reading from a .env file Pydantic will add the port to a redis url, so "redis://fakeredis" becomes "redis://fakeredis:6379". That breaks this check here, so the change is necessary to make this PR work. Another alternative way to do this could be to add the port to this string.

Reply:
Okay, let's make this more explicit by adding the port or, if for some reason the port might change, doing e.g. `re.fullmatch(f"redis://fakeredis:\d+", settings.redis_url)`